CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+2.1
+===========
+August 10, 2015
+
+Features
+--------
+* Added support for retry policies
+* Exposed raw paging state token via `cass_statement_set_paging_state_token()`
```

Comment:
I guess I was reading it wrong ... I was assuming you meant 'Exposed raw paging state token via `cass_result_paging_state_token()` and it can be added to a statement using `cass_statement_set_paging_state_token()`'